diff --git a/include/tdep-arm/libunwind_i.h b/include/tdep-arm/libunwind_i.h
index 291b101..66812b7 100644
--- a/include/tdep-arm/libunwind_i.h
+++ b/include/tdep-arm/libunwind_i.h
@@ -285,7 +285,7 @@ extern int tdep_access_fpreg (struct cursor *c, unw_regnum_t reg,
 			      unw_fpreg_t *valp, int write);
 
 /* unwinding method selection support */
-#define UNW_ARM_METHOD_ALL          0xFF
+#define UNW_ARM_METHOD_ALL          0xFD
 #define UNW_ARM_METHOD_DWARF        0x01
 #define UNW_ARM_METHOD_FRAME        0x02
 #define UNW_ARM_METHOD_EXIDX        0x04
